integration/test-graph.c

```diff
@@ -205,7 +205,7 @@ static inline int start_qemu_graph(struct branch *branch,
 
 
 #	define SSH(c) \
-	g_assert(!pg_util_ssh("localhost", ssh_port_id, glob_vm_key_path, c))
```

Comment:
ok, if you want a behavior where 0 is use as a successful value, g_assert should always assert that the value is NULL,
therefore this change is useless.